From c56c48d11d133e0a0d1ae9c1981cb92ea09732df Mon Sep 17 00:00:00 2001
From: Lim Siew Hoon <siew.hoon.lim@intel.com>
Date: Mon, 7 Oct 2024 14:20:00 +0800
Subject: [PATCH] msdk: adding back libdrm dependecy

upstream status: Inappropriate

Upstream revert libdrm dep from in sys/msdk, it put
in gst-plugins-bad directory folder to do libdrm_dep check.
Currently is put back libdrm dep in meson.build in sys/msdk.

Signed-off-by: Lim Siew Hoon <siew.hoon.lim@intel.com>
---
 sys/msdk/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sys/msdk/meson.build b/sys/msdk/meson.build
index 660ae130c..e22640ef6 100644
--- a/sys/msdk/meson.build
+++ b/sys/msdk/meson.build
@@ -185,6 +185,7 @@ if host_machine.system() == 'windows'
 else
   libdl_dep = cc.find_library('dl', required: get_option('msdk'))
   libgudev_dep = dependency('gudev-1.0', required: get_option('msdk'), allow_fallback: true)
+  libdrm_dep = dependency('libdrm', required: get_option('msdk'))
   msdk_deps = declare_dependency(dependencies: [gstva_dep, libdl_dep, libgudev_dep, libdrm_dep])
   msdk_deps_found = gstva_dep.found() and libdl_dep.found() and libgudev_dep.found() and libdrm_dep.found()
 endif
-- 
2.43.2

